beat-that-script.js

```diff
@@ -0,0 +1,149 @@
+//REQUIREMENTS//
+//1) There are two players and players take turns
+//2) when a player clicks submit, the game rolls 2 dice and shows the dice rolls
+//3) the player picks the order of the dice they want.
+//4) after both players have rolled and chosen the dice order, the game compares both players' scores. Player with the higher combined number wins
+
+// set global variables
+var rollDiceState = "rollDiceState";
+var chooseDiceState = "chooseDiceState";
+var compareScoresState = "compareScoresState";
```

Comment:
Good use of variables to assign each game state to a different variable. This prevents spelling mistakes when it's being used later on in the code.

However, since they are string values, you can spell then normally without pseudocode.
For e.g. at line 8 var rollDiceState = "roll dice state";